README.md

```diff
@@ -58,6 +58,37 @@ sudo make install
 # install fzf: https://github.com/junegunn/fzf
 ```
 
+### Using oh-my-zsh
+
+Make sure that your oh-my-zsh `ZSH_CUSTOM` directory is configured,
+then clone navi into the plugins directory.
+
+```sh
+export ZSH_CUSTOM='/path/to/.zsh'
+
+DST="$ZSH_CUSTOM/plugins"
```

Comment:
what is DST?

Reply:
`DST` stands for destination. It's a local variable (not exported) that is used to simplify the `make && cd` command. I can rename it to something else or inline if you'd prefer.

Reply:
could you rename it to `plugins_dir` or something like that?

Reply:
Sure